test/dialogflow-app-test.js

```diff
@@ -2184,6 +2184,24 @@ describe('DialogflowApp', function () {
     });
   });
 
+  /**
+   * Describes the behavior for DialogflowApp getIntent method.
+   */
+  describe('#getIntentName', function () {
+    // Success case test, when the API returns a valid 200 response with the response object
```

Comment:
Can we add a failure case test?

Reply:
Sure, but same here: I replicated the test from `getIntent` and adapted it to `getIntentName`.